MapboxMobileEventsTests/MMEEventsManagerTests.mm

```diff
@@ -962,7 +964,36 @@ @interface MMEEventsManager (Tests) <MMELocationManagerDelegate>
             });
         });
     });
-    
+
+    describe(@"- pushEvent:", ^{
+        beforeEach(^{
+            eventsManager.paused = YES;
+        });
+
+        context(@"when an error event is pushed", ^{
+            NSError* testError = [NSError.alloc initWithDomain:NSCocoaErrorDomain code:999 userInfo:@{
+                NSLocalizedDescriptionKey: @"Test Error Description",
+                NSLocalizedFailureReasonErrorKey: @"Test Error Failure Reason"
+            }];
+
+            [[MMEEventsManager sharedManager] pushEvent:[MMEEvent debugEventWithError:testError]];
```

Comment:
You’ll want to put this inside a `beforeEach`, `it`, or `subjectAction` block for a more expected execution order. Otherwise it will be run when the spec is loaded, which might be too early. Same thing below.